### ISSUE TITLE
The class Identifier - inconsistency between the definition and the property list, and between 8.9 and Annex A

### ISSUE THREAD

**Issue by jimjyang** [SEMICeu/DCAT-AP#334]
The definition of the class Identifier says "An identifier in a particular context, consisting of the string that is the identifier; an optional identifier for the identifier scheme; an optional identifier for the version of the identifier scheme; an optional identifier for the agency that manages the identifier scheme." 

1. The definition indicates that this class has one mandatory and three optional properties, while the specification has only one property, notation (skos:notation).  
2. In Annex A "Quick Reference of Classes and Properties" the property adms:identifier is Mandatory, but it is not listed/specified in 8.9. 
3. The definition of the property skos:notation (not adms:identifier) in 8.9 indicates that this is _the_ identifier (although the human-readable name is "notation"), however, the cardinality is 0..1 (not 1..1 if this is _the_ identifier, conf. the definition of the class). 

In other words, there is inconsistency between the definition and the property specifications, and between 8.9 and Annex A.

**Comment by bertvannuffelen** [comment]
@jimjyang thanks for pointing this out:

I rechecked DCAT-AP 2.1.1:

- skos:notation as property is mandatory and should thus have mincard 1. That is a mistake and has been corrected.
- The definition as it is now is indeed is prescribing a structure. In order to align with the latest release of adms I adopted the definition and usagenote of the class.
